api/v1alpha1/service_fence.proto

```diff
@@ -103,12 +106,23 @@ message ServiceFenceSpec {
     repeated string namespaceSelector = 3;
     // services match one selector of the label selector are all static dependency, will not expire
     repeated Selector labelSelector = 4;
+    // Criteria used to select the specific set of pods/VMs on which this
+    // `ServiceFence` configuration should be applied.
+    WorkloadSelector workload_selector = 5;
```

Comment:
The content in spec is generally for the user to configure, not for the program to do data caching. I suggest that the selector data of the svc that ServiceFence is going to use is either taken in real time when generating the sidecar, or sensed and stored in status when the svc changes.

Reply:
sf.spec.workloadSelector.fromService = true, sidecar.workloadSelector = svc.spec.selector

```yaml
#fence
  spec:
    enable: true
    workloadSelector:
      fromService: true